paper-menu-button.html

```diff
@@ -79,28 +79,31 @@
         @apply(--paper-menu-button-disabled);
       }
 
-      :host([vertical-align="top"]) paper-material {
-        margin-bottom: 20px;
-        margin-top: -10px;
-        top: 10px;
-      }
-
-      :host([vertical-align="bottom"]) paper-material {
-        bottom: 10px;
-        margin-bottom: -10px;
-        margin-top: 20px;
-      }
-
       iron-dropdown {
         @apply(--paper-menu-button-dropdown);
       }
 
       .dropdown-content {
+        @apply(--shadow-elevation-2dp);
```

Comment:
Oh, I didn't think about that, Shady/Shadow CSS encapsulation should handle this but I suppose it would be less confusing to make this `.dropdown-content-container` (or similar). Is the concern that they might try and style `.dropdown-content` in a stylesheet that isn't picked up by Shady DOM's style rewriter?